src/Network/Api/Postmark/Data.hs

```diff
@@ -98,9 +113,14 @@ instance ToJSON Email where
     , oljson "Cc" (emailCc v) (T.intercalate ",")
     , oljson "Bcc" (emailBcc v) (T.intercalate ",")
     , omjson "Headers" (emailHeaders v)
+    , ojson "TrackOpens" (emailTrackOpens v)
+    , ojson "TrackLinks" (emailTrackLinks v)
     , oljson "Attachments" (emailAttachments v) id
     ])
 
+instance ToJSON TrackLinks where
+  toJSON = String . pack . show
```

Comment:
I understand the desire here, but I would prefer to be more explicit here, if someone comes along and adds a constructor it is to easy to miss that the name matters for serialisation. If you do a total pattern match here, you will get a compile error when someone adds a constructor and it will be obvious how to do the right thing.